### ISSUE TITLE
Support for federated evaluation

### ISSUE THREAD

**Issue by mansishr** [intel/openfl#788]
Add an example of federated evaluation using [aggregator based workflow](https://openfl.readthedocs.io/en/latest/running_the_federation.html#aggregator-based-workflow) or the [director based workflow](https://openfl.readthedocs.io/en/latest/running_the_federation.html#director-based-workflow). Federated evaluation refers to model evaluation on the client/collaborator side. Once the global model is sent to the collaborators, they could perform one pass of local evaluation. Evaluation wouldn't need any training/construction of optimizers/gradient descent.

For aggregator based workflow:
- An existing workspace like `torch_cnn_mnist` could be taken as a base example from the list of [supported templates for OpenFL](https://github.com/securefederatedai/openfl/tree/develop/openfl-workspace) to create a new template for federated evaluation. 

- Changes to the `plan/plan.yaml` will be needed:
  - Since there's no training involved, parameter `rounds_to_train` should be set to 1
  - Task `aggregator_model_validation` should be assigned instead of the [default tasks](https://github.com/securefederatedai/openfl/blob/develop/openfl-workspace/workspace/plan/defaults/tasks_torch.yaml).

- Changes to the `src/pt_cnn.py` will be needed:
  - No optimizer initialization needed.

**Comment by bjklemme-intel** [comment]
Could you supply more detail on what you are looking for?

**Comment by bjklemme-intel** [comment]
this is a test comment and will be immediately deleted

**Comment by bjklemme-intel** [comment]
this is a test

**Comment by wenjiewang-intel** [comment]
OpenFL_Team4 is working on this
